src/classes/OpportunityTriggerHandler.cls

```diff
@@ -0,0 +1,282 @@
+                                    }
+                                    If(str.startsWith('00:11:CE:D')) {
+                                        Automobile_Presence_Sensor__c autoPSensor = (Automobile_Presence_Sensor__c)oldMapOfAutomobileRecords.get(str);
+                                        autoPSensor.Account__c = opp.Shipment_Account__c;
+                                        updateListOfSensors.add(autoPSensor);
+                                        content += '<tr><td>Automobile Presence Sensor</td><td>' + str + '</td><td>Sensor moved from installation account to Shipment account : ' + opp.Shipment_Account__c + '</td></tr>';
+                                    }
+                                    If(str.startsWith('00:11:CE:E')) {
+                                        Automobile_TDU__c autoTDUSensor = (Automobile_TDU__c)oldMapOfAutomobileRecords.get(str);
+                                        autoTDUSensor.Account__c = opp.Shipment_Account__c;
+                                        updateListOfSensors.add(autoTDUSensor);
+                                        content += '<tr><td>Automobile TDU Sensor</td><td>' + str + '</td><td>Sensor moved from installation account to Shipment account : ' + opp.Shipment_Account__c + '</td></tr>';
+                                    }
+                                    If(str.subString(1,3).isNumeric()) {
+                                        Automobile_Tag__c autoTagSensor = (Automobile_Tag__c)oldMapOfAutomobileRecords.get(str);
+                                        autoTagSensor.Account__c = opp.Shipment_Account__c;
+                                        updateListOfSensors.add(autoTagSensor);
+                                        content += '<tr><td>Automobile Tag Sensor</td><td>' + str + '</td><td>Sensor moved from installation account to Shipment account : ' + opp.Shipment_Account__c + '</td></tr>';
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+                content = content + '</table>'+'<br/>';
+                string content2 = '<b>Unsuccessful:</b><br><table border="1" cellpadding=5 cellspacing=0>' + '<tr><th>Automobile Type</th><th>Name</th><th>Message</th></tr>';
+                //Unsuccessful
+                if(!setOfAutomobileTags.isEmpty()) {
+                    //content2 += '<b> Unsuccessful:</b> <table border="1" cellpadding=5 cellspacing=0>' + '<tr><th>Automobile Type</th><th>Name</th> <th>Message</th></tr>';
+                    for(String tag : setOfAutomobileTags) {
+                        if(!tag.startsWith('00:11:CE:00') && !tag.startsWith('00:11:CE:D') && !tag.startsWith('00:11:CE:E') && !tag.substring(0,3).isNumeric()
+                            && (opp.StageName == 'Prospect' || opp.StageName == 'Qualification' || opp.StageName=='Processing' || opp.StageName=='Closed Lost'
```

Comment:
why added this -> 
 opp.StageName == 'Prospect' || opp.StageName == 'Qualification' || opp.StageName=='Processing' || opp.StageName=='Closed Lost'
 
 There no need